packages/bbui/src/Markdown/MarkdownEditor.svelte

```diff
@@ -43,7 +45,7 @@
   const debouncedUpdate = debounce(update, 250)
 </script>
 
-{#key height}
+{#key (height, readonly)}
```

Comment:
We don't key using `disabled` which is used in the same way, so I'm assuming we either *do* need to key by disabled (if it does not toggle properly) or *do not* need to key by `readonly`.

Reply:
Nice catch - was able to remove the readonly key.